erpnext/accounts/doctype/coupon_code/test_records.json

```diff
@@ -0,0 +1,70 @@
+[
```

Comment:
test_records.json style is deprecated (avoid!)

Reply:
Sure, test_records.json is removed